helm/profiles.yaml

```diff
@@ -108,3 +126,74 @@ daskhub:
                   operator: NotIn
                   values:
                   - gpu
+
+      extraFiles:
+        spark_default_configuration:
+          # TODO(https://github.com/hashicorp/terraform-provider-helm/issues/628): use set-file
+          stringData: |
+            """
+            Default Spark configuration init for the Jypyter instance.
+            """
+            import socket
+            import os
+            notebook_ip = socket.gethostbyname(socket.gethostname())
+            namespace_user = os.environ.get('NAMESPACE_USER', '')
+            spark_config = {
+                'spark.master': 'k8s://https://kubernetes.default.svc.cluster.local',
+                'spark.app.name': 'STAC API with RF in K8S',
```

Comment:
The Spark app name should probably be picked up from the Notebook name / or used the default one.